services/bettersocial/chat/updateBetterChannelMembers.js

```diff
@@ -69,7 +70,8 @@ const updateBetterChannelMembers = async (
   return {
     betterChannelMember: better_channel_member,
     newChannelName: checkedChannelName,
-    betterChannelMemberObject: _.keyBy(better_channel_member, 'user_id')
+    betterChannelMemberObject: _.keyBy(better_channel_member, 'user_id'),
+    updatedChannel: channel
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The addition of `updatedChannel: channel` to the return object is a good change. It ensures that the updated state of the channel is returned by the function and can be used elsewhere in the application. This will help in maintaining the latest state of the channel throughout the application.


<!-- This is an auto-generated comment by OSS CodeRabbit -->